src/renderer/util/dom.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import * as React from 'react';
+
+// is a child component being rendered?
+export const isShowingChildren = (children: React.Node) => {
+  if (Array.isArray(children)) {
+    for (let i = 0; i < children.length; i += 1) {
```

Comment:
I'm guessing this code was copy/pasted, but `children.findIndex()` would make this code both more succinct and reduce state.